server.js

```diff
@@ -0,0 +1,16 @@
+import express from "express";
+
+const app = express();
+const port = 3001;
+
+app.get("/", (req, res) => {
+	res.send("Hello World!");
+});
+
+app.get("/api/todos", (req, res) => {
+	res.json([{ id: 1, name: "Uwe" }]);
+});
+
+app.listen(port, () => {
+	console.log(`Test`);
```

Comment:
I think a more descriptive message would be more appropriate, something actually worked, it's not a test